lib/cloudprint/printer.rb

```diff
@@ -25,6 +25,13 @@ def print(options)
       client.print_jobs.new_from_response response["job"]
     end
 
+    def all_jobs
+      method = :post
+      response = client.connection.send(method, '/jobs', :printerid => self.id) || {}
+      return nil if response.nil? || response['jobs'].nil?
```

Comment:
Two things here:

1. `response` can't ever be nil if we're doing `|| {}` at the end of line 30
2. I'm not sure I like returning nil here, since this is a method called all_jobs. Either we think this is an exceptional situation and we raise, or we return an empty array.